test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -143,6 +142,125 @@ function GetTestData
     return $body
 }
 
+<#
+    Extract the raw content from a response and
+    convert the embedded json to PSObjects.
+    Assumes the raw content is from the  HTTPListener.
+#>
+function Convert-ResponseRawContent
+{
+    param
+    (
+        [Parameter(Mandatory)]
+        [ValidateNotNull()]
+        [Microsoft.PowerShell.Commands.WebResponseObject] $Response
+    )
+
+    $index = $Response.RawContent.IndexOf("{")
```

Comment:
why isn't $response.content | convertfrom-json sufficient?  this code seems unnecessary